Logging started at 2015-10-28 17:21:29
File name: C:\Users\root\Dropbox\Vencillio Advanced\VA Source\.\data\logs\err\1446049289124 - root - [Wed Oct 28][5.21PM].txt
System.getProperty("os.name"): "Windows Server 2008 R2"

[2015-10-28 17:27:58] com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown column 'username' in 'where clause'
[2015-10-28 17:27:58] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2015-10-28 17:27:58] 	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
[2015-10-28 17:27:58] 	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
[2015-10-28 17:27:58] 	at java.lang.reflect.Constructor.newInstance(Unknown Source)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.Util.getInstance(Util.java:381)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3515)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3447)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1951)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2101)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2554)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1761)
[2015-10-28 17:27:58] 	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1912)
[2015-10-28 17:27:58] 	at com.vencillio.core.network.mysql.ExternalDatabase$InteralQuery.call(ExternalDatabase.java:224)
[2015-10-28 17:27:58] 	at com.vencillio.core.network.mysql.ExternalDatabase$InteralQuery.call(ExternalDatabase.java:1)
[2015-10-28 17:27:58] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2015-10-28 17:27:58] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2015-10-28 17:27:58] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2015-10-28 17:27:58] 	at java.lang.Thread.run(Unknown Source)
[2015-10-28 17:27:58] java.lang.NullPointerException
[2015-10-28 17:27:58] 	at com.vencillio.core.network.mysql.VoteUpdater.lambda$0(VoteUpdater.java:53)
[2015-10-28 17:27:58] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2015-10-28 17:27:58] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2015-10-28 17:27:58] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2015-10-28 17:27:58] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2015-10-28 17:27:58] 	at java.lang.Thread.run(Unknown Source)
[2015-10-28 17:28:11] java.lang.ArrayIndexOutOfBoundsException: 80
[2015-10-28 17:28:11] 	at com.vencillio.rs2.content.interfaces.impl.AchievementTab.<init>(AchievementTab.java:47)
[2015-10-28 17:28:11] 	at com.vencillio.rs2.content.achievements.AchievementHandler.activateAchievement(AchievementHandler.java:40)
[2015-10-28 17:28:11] 	at com.vencillio.rs2.content.vencilliobot.VencillioBot.answered(VencillioBot.java:127)
[2015-10-28 17:28:11] 	at com.vencillio.rs2.content.vencilliobot.VencillioBot.answer(VencillioBot.java:106)
[2015-10-28 17:28:11] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:84)
[2015-10-28 17:28:11] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2015-10-28 17:28:11] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2015-10-28 17:28:11] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2015-10-28 17:28:11] 	at com.vencillio.rs2.entity.World.process(World.java:319)
[2015-10-28 17:28:11] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2015-10-28 17:28:11] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2015-10-28 17:28:38] java.lang.NumberFormatException: For input string: "500k"
[2015-10-28 17:28:38] 	at java.lang.NumberFormatException.forInputString(Unknown Source)
[2015-10-28 17:28:38] 	at java.lang.Integer.parseInt(Unknown Source)
[2015-10-28 17:28:38] 	at java.lang.Integer.parseInt(Unknown Source)
[2015-10-28 17:28:38] 	at com.vencillio.rs2.entity.player.net.in.impl.StringInputPacket.handle(StringInputPacket.java:41)
[2015-10-28 17:28:38] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2015-10-28 17:28:38] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2015-10-28 17:28:38] 	at com.vencillio.rs2.entity.World.process(World.java:319)
[2015-10-28 17:28:38] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2015-10-28 17:28:38] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2015-10-28 17:28:44] java.lang.ArrayIndexOutOfBoundsException: 80
[2015-10-28 17:28:53] 	at com.vencillio.rs2.content.interfaces.impl.AchievementTab.<init>(AchievementTab.java:47)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.content.achievements.AchievementHandler.activateAchievement(AchievementHandler.java:40)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.content.consumables.Consumables.consume(Consumables.java:116)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:822)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.entity.World.process(World.java:319)
[2015-10-28 17:28:53] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2015-10-28 17:28:53] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2015-10-28 17:28:53] java.lang.ArrayIndexOutOfBoundsException: 80
[2015-10-28 17:28:53] 	at com.vencillio.rs2.content.interfaces.impl.AchievementTab.<init>(AchievementTab.java:47)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.content.achievements.AchievementHandler.activateAchievement(AchievementHandler.java:40)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.content.consumables.Consumables.consume(Consumables.java:116)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:822)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2015-10-28 17:28:53] 	at com.vencillio.rs2.entity.World.process(World.java:319)
[2015-10-28 17:28:53] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2015-10-28 17:28:53] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2015-10-28 17:29:26] java.lang.ArrayIndexOutOfBoundsException: 80
[2015-10-28 17:29:26] 	at com.vencillio.rs2.content.interfaces.impl.AchievementTab.<init>(AchievementTab.java:47)
[2015-10-28 17:29:26] 	at com.vencillio.rs2.content.achievements.AchievementHandler.activateAchievement(AchievementHandler.java:40)
[2015-10-28 17:29:26] 	at com.vencillio.rs2.content.consumables.Consumables.consume(Consumables.java:116)
[2015-10-28 17:29:26] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:822)
[2015-10-28 17:29:26] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2015-10-28 17:29:26] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2015-10-28 17:29:26] 	at com.vencillio.rs2.entity.World.process(World.java:319)
[2015-10-28 17:29:26] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2015-10-28 17:29:26] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2015-10-28 17:30:54] java.lang.ArrayIndexOutOfBoundsException: 80
[2015-10-28 17:30:54] 	at com.vencillio.rs2.content.interfaces.impl.AchievementTab.<init>(AchievementTab.java:47)
[2015-10-28 17:30:54] 	at com.vencillio.rs2.content.achievements.AchievementHandler.activateAchievement(AchievementHandler.java:40)
[2015-10-28 17:30:54] 	at com.vencillio.rs2.content.shopping.Shop.buy(Shop.java:275)
[2015-10-28 17:30:54] 	at com.vencillio.rs2.content.shopping.Shopping.buy(Shopping.java:58)
[2015-10-28 17:30:54] 	at com.vencillio.rs2.entity.player.net.in.impl.ChatInterfacePacket.handleEnterX(ChatInterfacePacket.java:126)
[2015-10-28 17:30:54] 	at com.vencillio.rs2.entity.player.net.in.impl.ChatInterfacePacket.handle(ChatInterfacePacket.java:39)
[2015-10-28 17:30:54] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2015-10-28 17:30:54] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2015-10-28 17:30:54] 	at com.vencillio.rs2.entity.World.process(World.java:319)
[2015-10-28 17:30:54] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2015-10-28 17:30:54] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
